### ISSUE TITLE
ETK mangles Inferlink date extraction

### ISSUE THREAD

**Issue by szeke** [usc-isi-i2/dig-etl-engine#55]
Inferlink extracted the text `1892` and etk converted it `1992`, which is the value in the `value` attributes:

```
"date_creation-0-2": {
          "simple_tokens_original_case": [
            "1892"
          ],
          "simple_tokens": [
            "1892"
          ],
          "text": "1892",
          "data_extraction": {
            "date_creation": {
              "extract_from_landmark": {
                "results": [
                  {
                    "origin": {
                      "segment": "html",
                      "document_id": "9B633E9FBEDEDBA1CF8119B3DBB9A2111034787C5CC130DF91366FA6F3375E67"
                    },
                    "value": "1992-01-08T00:00:00"
                  }
                ]
              }
            }
          }
        }
```

**Comment by szeke** [comment]
[museum.zip](https://github.com/usc-isi-i2/dig-etl-engine/files/1414443/museum.zip)

Attached museum project folder, which should support reproducing the problem.

**Comment by saggu** [comment]
Add strict_parse as flag to etk config. Right now its set to true by default in core.py

**Comment by szeke** [comment]
Closing as the bug is diagnosed and needs to be fixed with new features in #102
